docs/_templates/sidebarlogo.html

```diff
@@ -50,30 +50,31 @@ <h3>Other Projects</h3>
   <li><a href="https://pipenv-pipes.readthedocs.io/en/latest/">Pipenv-Pipes</a></li>
 </ul>
 
-<p>More projects founded by <a href="http://kennethreitz.org/">Kenneth Reitz</a>:</p>
+<p>More projects founded by <a href="https://kenreitz.org">Kenneth Reitz</a>:</p>
 <ul>
-    <li><a href="http://pep8.org/">pep8.org</a></li>
-    <li><a href="http://httpbin.org/">httpbin.org</a></li>
+    <li><a href="https://pep8.org/">pep8.org</a></li>
+    <li><a href="https://httpbin.org/">httpbin.org</a></li>
     <li><a href="https://howtopython.org">HowToPython.org</a></li>
-    <li><a href="http://python-guide.org">The Python Guide</a></li>
-    <li><a href="http://python-requests.org/">Requests: HTTP for Humans</a></li>
-    <li><a href="https://github.com/kennethreitz/records">Records: SQL for Humans</a></li>
-    <li><a href="https://frostming.github.io/legit">Legit: Git for Humans</a></li>
+    <li><a href="https://python-guide.org">The Python Guide</a></li>
+    <li><a href="https://requests.readthedocs.io">Requests: HTTP for Humans</a></li>
+    <li><a href="https://github.com/timofurrer/maya">Maya: Datetimes for Humans</a></li>
+    <li><a href="https://github.com/kennethreitz-archive/records">Records: SQL for Humans</a></li>
+    <li><a href="https://frostming.github.io/legit/">Legit: Git for Humans</a></li>
     <li><a href="http://docs.python-tablib.org/en/latest/">Tablib: Tabular Datasets</a></li>
 </ul>
 
 
 <h3>Useful Links</h3>
 <ul>
-  <li><a href="http://docs.python-requests.org/en/latest/community/recommended/">Recommended Packages and Extensions</a></li>
+  <li><a href="https://requests.readthedocs.io/en/latest/community/recommended/">Recommended Packages and Extensions</a></li>
 
   <p></p>
 
-  <li><a href="http://github.com/pypa/pipenv">Pipenv @ GitHub</a></li>
-  <li><a href="http://pypi.org/project/pipenv">Pipenv @ PyPI</a></li>
+  <li><a href="httsp://github.com/pypa/pipenv">Pipenv @ GitHub</a></li>
```

Comment:
Typo

Reply:
Ah 🤦‍♂️ Good catch!  Pushed and updated accordingly.